src/app/pages/CpsSty/index.jsx

```diff
@@ -112,10 +152,35 @@ const CpsStyContainer = ({ pageData }) => {
         aboutTags={aboutTags}
       />
       <ATIAnalytics data={pageData} />
-      <StyledGhostGrid as="main" role="main">
-        <Blocks blocks={blocks} componentsToRender={componentsToRender} />
-      </StyledGhostGrid>
-      <CpsRelatedContent content={relatedContent} />
+
+      <StyledGrid columns={gridColumns} enableGelGutters margins={gridMargins}>
+        <Grid
+          item
+          columns={gridColsMain}
+          startOffset={gridOffset}
+          as="main"
+          role="main"
+        >
+          <Blocks blocks={blocks} componentsToRender={componentsToRender} />
+          <CpsRelatedContent content={relatedContent} />
+        </Grid>
+        <Grid item columns={ColSecondaryLayout}>
```

Comment:
We could maybe consider moving this `Grid item` to `src/app/components/Grid/index.jsx` and reusing it, rather than importing a layout to be reused? It could be a [higher order component](https://reactjs.org/docs/higher-order-components.html) like this:
```javascript
const SecondColumnItem({children}) => {
  export const ColumnWrapper = styled.div`
    margin-top: ${GEL_SPACING_QUAD};
  `;

  const ColSecondaryLayout = {
    group0: 8,
    group1: 8,
    group2: 8,
    group3: 8,
    group4: 3,
    group5: 3,
  };

  return <Grid item columns=ColSecondaryLayout>
    <ColumnWrapper>
     {children}
    </ColumnWrapper>
  </Grid>
} 
```

Maybe this is too rigid though? Just thinking it maybe a bit hard to understand in the future how you would use this to implement a second column without the context: https://github.com/bbc/simorgh/pull/5417/files#diff-860eef0af93e58fa85364ccd46280087R65 ?

Reply:
We could also consider creating a `FirstColumnItem` alongside this one so you could see how the column configurations relate to each other:
```
    group0: 8,
    group1: 8,
    group2: 8,
    group3: 8,
    group4: 7,
    group5: 5,
```

Reply:
Or finally create a fully fledged `TwoColumnLayout` HOC that could be reused across pages, I'll 🤐 now!

Reply:
I think you're right about keeping the styling in the `src/app/pages/CpsSty/index.jsx` and even the column layout @andrewscfc . The HOC is a great idea and I think that what we want to end up with, but for now it might be best to think about re-usability after we have finished all the necessary work to get this off-forge. 

If we have time after we've done what we need for STY and PGL pages maybe then we can think more about the re-usability as we don't really know the full requirements for Article and other page types that might use a secondary column layout at the moment. It might be beneficial to create a separate issue to make sure we don't forget about this when it comes to working on it in the future. What do you think?

Reply:
I think we if we don't implement a `TwoColumnLayout` component or similar we'd be best to keep the layout logic all in `src/app/pages/CpsSty/index.jsx`:
https://github.com/bbc/simorgh/pull/5417/files#diff-860eef0af93e58fa85364ccd46280087R61-R73
I'm not sure of the benefit of extracting the layout config to `src/app/components/Grid/index.jsx` at this stage?

Leaving the layout config in `src/app/pages/CpsSty/index.jsx` allows maintainers to see the how it is being used to deliver a STY page. I think it would be good to create an issue for a `TwoColumnLayout` component @HarveyPeachey so it is visible that we have a two column layout ready to be extracted into a shared component.

Reply:
Might be worth confirming with UX if Articles actually require a 2nd column, I've never seen one in designs...